### PR TITLE
MAT-4296

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-4296](https://jira.cms.gov/browse/MAT-4296)
Jira Ticket: [MAT-4597](https://jira.cms.gov/browse/MAT-4597)

Needs to be merged along side madie-measure pr
https://github.com/MeasureAuthoringTool/madie-measure/pull/239

(Optional) Related Tickets:

* This PR updates the edit measure page, listens for broadcasted events from layout to delete measures
### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/PageHeader/PageHeader.tsx [hunk 1]
```diff
@@ -61,8 +72,23 @@ const PageHeader = () => {
...
+                  const event = new Event("delete-measure");
+                  window.dispatchEvent(event);
+                }}
+              >
+                <DeleteOutlineOutlinedIcon
+                  className="page-header-action-icon"
+                  fontSize="small"
+                />
+                <div>Delete Measure</div>
+              </button>
```
Comment (RohitKandimalla) [inline src/components/PageHeader/PageHeader.tsx:91]: Why aren't we using button from Madie-design-system ?

Comment (RohitKandimalla) [reply src/components/PageHeader/PageHeader.tsx:91]: I thought the idea of variant="action" is to handle these types of buttons

Comment (mcmcphillips) [reply src/components/PageHeader/PageHeader.tsx:91]: I think I need to make another change to the design system to make it handle icons correctly.
I can make that change when I'm back tuesday

Comment (mcmcphillips) [reply src/components/PageHeader/PageHeader.tsx:91]: You can see the variant=action used for the discard changes button on the edit pages for the current madie-measure pull request

**Review by RohitKandimalla — APPROVED** [review]
